src/test/java/com/puppycrawl/tools/checkstyle/checks/UniquePropertiesCheckTest.java

```diff
@@ -197,17 +197,15 @@ public void testWrongKeyTypeInProperties() throws Exception {
      * Method generates FileNotFound exception details. It tries to open file,
```

Comment:
Javadoc should change. `FileNotFound exception` is not generated anymore.

Reply:
done